src/Module.php

```diff
@@ -160,24 +156,23 @@ public function getServiceConfig()
                 ReportInterface::class => Report::class,
 
                 // Legacy Zend Framework aliases
-                'ZendDeveloperTools\ReportInterface'    => ReportInterface::class,
-                \ZendDeveloperTools\Report::class       => Report::class,
-                'ZendDeveloperTools\ExceptionCollector' => 'Laminas\DeveloperTools\ExceptionCollector',
-                'ZendDeveloperTools\RequestCollector'   => 'Laminas\DeveloperTools\RequestCollector',
-                'ZendDeveloperTools\ConfigCollector'    => 'Laminas\DeveloperTools\ConfigCollector',
-                'ZendDeveloperTools\MailCollector'      => 'Laminas\DeveloperTools\MailCollector',
-                'ZendDeveloperTools\MemoryCollector'    => 'Laminas\DeveloperTools\MemoryCollector',
-                'ZendDeveloperTools\TimeCollector'      => 'Laminas\DeveloperTools\TimeCollector',
-                'ZendDeveloperTools\FlushListener'      => 'Laminas\DeveloperTools\FlushListener',
-                \ZendDeveloperTools\Profiler::class     => Profiler::class,
-                'ZendDeveloperTools\Config'             => 'Laminas\DeveloperTools\Config',
-                'ZendDeveloperTools\Event'              => 'Laminas\DeveloperTools\Event',
-                'ZendDeveloperTools\StorageListener'    => 'Laminas\DeveloperTools\StorageListener',
-                ToolbarListener::class                  => Listener\ToolbarListener::class,
-                ProfilerListener::class                 => Listener\ProfilerListener::class,
-                EventLoggingListenerAggregate::class
-                    => Listener\EventLoggingListenerAggregate::class,
-                'ZendDeveloperTools\DbCollector' => 'Laminas\DeveloperTools\DbCollector',
+                'ZendDeveloperTools\ReportInterface'                        => ReportInterface::class,
+                'ZendDeveloperTools\Report'                                 => Report::class,
+                'ZendDeveloperTools\ExceptionCollector'                     => 'Laminas\DeveloperTools\ExceptionCollector',
```

Comment:
The Laminas classes on the RHS can all be `::class` references no?

Reply:
Those classes don't actually exist. I'm guessing they were previously used names? As they're defined below in the invokables section